test/cli/apps/create.test.js

```diff
@@ -247,46 +248,19 @@ describe('eg apps create', () => {
     assert.equal(usage, 'must include --stdin or -u, --user');
   });
 
-  it('prints error on invalid JSON from stdin', done => {
```

Comment:
why it is removed ?

Reply:
I can put this back if you want, but basically this is testing something that now is handled by JSON Schema library (ajv) — and it's guaranteed to work. Previously it made sense, because it was handled with custom code, but now I think this can be safely removed.

In particular, an invalid JSON will _never_ pass any schema validation. So you'll get an error, no doubts on that.

Reply:
what if I accidentally disable the code that does validation ? 
previous tests are positive, so I will never know that validation logic is missing, right ?

Reply:
> what if I accidentally disable the code that does validation ?
In that case, I suppose, a lot of other tests will break.

I've followed the code paths done by these two tests and they're covered by  a lot of other tests.

You can also see that although I removed two tests, the code coverage stayed the same (actually I think it increased).

I will readd these tests, but I feel they're not adding any new value.